### PR TITLE
chore(testing): increase coverage on Attribute

### LINKED ISSUE #2551: [Attribute] improve unit test coverage

### PR DESCRIPTION
**Pull request by kevinsperrine**
Closes #2551 

**Summary**

- add Attribute.test.jsx and target specific missing portions

**Change List (commits, features, bugs, etc)**

- new Attribute.test.jsx

**Acceptance Test (how to verify the PR)**

- test coverage 90%+

**Regression Test (how to make sure this PR doesn't break old functionality)**

- n/a

**Things to look for during review**

- [ ] Make sure all references to `iot` or `bx` class prefix is using the prefix variable
- [ ] (React) All major areas have a `data-testid` attribute. New test ids should have test written to ensure they are not changed or removed.
- [ ] UI should be checked in RTL mode to ensure the proper handling of layout and text.
- [ ] All strings should be translatable.
- [ ] The code should pass a11y scans (The storybook a11y knob should show no violations). New components should have a11y test written.
- [ ] Unit test should be written and should have a coverage of 90% or higher in all areas.
- [ ] All components should be passing visual regression test. For new styles or components either a visual regression test should be written for all permutations or the base image updated.
- [ ] Changes or new components should either write new or update existing documentation.
- [ ] PR should link and close out an existing issue

### CODE REVIEW

File: packages/react/src/components/ValueCard/Attribute.test.jsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+      expect.stringContaining(
+        'Invalid prop `attribute.thresholds[0].comparison` of value `===` supplied to `Attribute`'
+      )
+    );
+    console.error = error;
+  });
+
+  it('should not render a trend icon when an invalid trend is passed to secondaryValue', () => {
+    const { error } = console;
+    console.error = jest.fn();
```
Comment (bjornalm) [inline packages/react/src/components/ValueCard/Attribute.test.jsx:80]: nit: 
You could also write `jest.spyOn(console, 'error');` and later  `console.error.mockRestore();`

**Review by bjornalm — APPROVED** [review]
Great work

**Review by davidicus — APPROVED** [review]

File: packages/react/src/components/ValueCard/Attribute.test.jsx (hunk 1 above)
Comment (kevinsperrine) [reply packages/react/src/components/ValueCard/Attribute.test.jsx:80]: I did it this way, so the errors stays hidden when the tests are running to keep the output cleaner.

Comment (bjornalm) [reply packages/react/src/components/ValueCard/Attribute.test.jsx:80]: That makes sense, like using `jest.spyOn(console, 'error').mockImplementation(() => {});`

Comment (kevinsperrine) [reply packages/react/src/components/ValueCard/Attribute.test.jsx:80]: Yep. more than one way to skin a cat.

**Pull request merged**